TestInDebugMode.test_start_with_number_call_is_ignored fails in CI

Bug Report
==========
<!--
Please fill in a a meaningful and concise title in the field above.
It helps us to classify the issue even  before reading the body and handling the
issue a bit better.

Examples:
  * Great: "Unable to select fleet in galaxy map"
  * Good: "Unable to click fleet"
  * Not really okay: "Issue with fleets"
  * Horrible: "HELP!!!"
-->


Environment
-----------
<!--
You don't need to provide the information in section when they are not useful in
the context of this issue.  For example a crashing hosted server may not need
information about the graphic card, but maybe depend on the operating system you
use.  When in doubt provide all information you have available.

You can find the FreeOrion version number in the lower right corner of the game
main screen.

When stating the operating system also note the version of the operating system
so, `Windows 8 Pro` instead of `Windows` or `Mac OSX Mountain Lion` instead of
`OSX`.
-->

* **FreeOrion Version**: PRs rebased on master (fec33c280c891195c6e8e481ba3b91c6f99a8f1a)
* **Operating System**: Ubuntu
* **Fetched as** <!-- delete lines below that don't apply -->
  * Compiled from source


Description
-----------
<!--
Add a meaningful description of the bug you encountered.

If a screenshot or image helps to describe the issue content feel free to create
one.

Also attach the log files the game creates.  The log files The log files can be
found on
* Windows: %APPDATA%\FreeOrion
* MacOSX: ~/Library/"Application Support"/FreeOrion
* Linux: ${XDG_DATA_HOME:-~/.local/share}/freeorion
-->
Something changed either in CI or in master and Python test fails with Python 3.10.

```
=================================== FAILURES ===================================
____________ TestInDebugMode.test_start_with_number_call_is_ignored ____________

self = <test_debug_chat.TestInDebugMode object at 0x7f2f6c594d00>
debug_handler = <freeorion_tools.chat_handler.debug_chat_handler.DebugChatHandler object at 0x7f2f6c33fb50>

    def test_start_with_number_call_is_ignored(self, debug_handler: DebugChatHandler):
        assert debug_handler.process_message(1, "start 2")
        chat, log = get_output_from_handler(debug_handler)
>       assert "SyntaxError: invalid syntax" in log
E       assert 'SyntaxError: invalid syntax' in '>>> start 2\n  File "<input>", line 1\n    start 2\n          ^\nSyntaxError: incomplete input\n'

chat       = ('Entering debug mode\n'
 'Print stop to exit.\n'
 'Local variables:\n'
 '  variable_adescription for a\n'
 '  variable_bdescription for b\n'
 '  variable_cvariable with import\n'
 '  variable_a_plus_bdescription for a+b\n'
 '  File "<input>", line 1\n'
 '    start 2\n'
 '          ^\n'
 'SyntaxError: incomplete input\n')
debug_handler = <freeorion_tools.chat_handler.debug_chat_handler.DebugChatHandler object at 0x7f2f6c33fb50>
log        = ('>>> start 2\n'
 '  File "<input>", line 1\n'
 '    start 2\n'
 '          ^\n'
 'SyntaxError: incomplete input\n')
self       = <test_debug_chat.TestInDebugMode object at 0x7f2f6c594d00>

default/python/tests/AI/test_debug_chat.py:254: AssertionError
=========================== short test summary info ============================
FAILED default/python/tests/AI/test_debug_chat.py::TestInDebugMode::test_start_with_number_call_is_ignored
======================== 1 failed, 198 passed in 0.97s =========================
Error: Process completed with exit code 1.
```

Expected Result
---------------
<!--
When you enter an issue please add a description of what behaviour you would
expect from the game instead of the issue.
-->
CI passes

Steps to reproduce
------------------
<!--
If the error only manifests itself after doing a certain number of actions
please add a list of steps to reproduce the bug.  If the error only occurs in
a certain game state please be sure to add a save game to the issue.

To upload save games please pack them into a Zip archive (*.zip) first.
-->
* Change any Python file.
* Push in PR.

--- a/default/python/tests/AI/test_debug_chat.py
+++ b/default/python/tests/AI/test_debug_chat.py
@@ -251,8 +251,8 @@ class TestInDebugMode:
     def test_start_with_number_call_is_ignored(self, debug_handler: DebugChatHandler):
         assert debug_handler.process_message(1, "start 2")
         chat, log = get_output_from_handler(debug_handler)
-        assert "SyntaxError: invalid syntax" in log
-        assert "SyntaxError: invalid syntax" in chat
+        assert "SyntaxError" in log
+        assert "SyntaxError" in chat
 
 
 class TestRestart:

